test/views/withdraw/WithdrawConfirmSpec.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.withdraw
+
+import controllers.routes
+import org.jsoup.Jsoup
+import play.api.i18n.Messages.Implicits._
+import testHelpers.ViewSpecHelpers.{CommonMessages, CommonViewSpecHelper}
+import testHelpers.ViewSpecHelpers.withdraw.WithdrawConfirmSpecMessages
+import views.html.pages.withdraw.{withdrawConfirm => views}
+
+class WithdrawConfirmSpec extends CommonViewSpecHelper with WithdrawConfirmSpecMessages {
+
+  "Withdraw Confirm view" when {
+    lazy val withdrawDate = "16/01/2018"
+    lazy val view = views(withdrawDate, "ip14", "dormant")
+    lazy val doc = Jsoup.parse(view.body)
+
+    s"have a title ${"pla.withdraw.what-happens.info-heading"}" in {
+      doc.title() shouldBe plaWithdrawWhatHappensInfoHeading
+    }
+
+    s"have a question of ${"pla.withdraw.what-happens.info-heading"}" in {
+      doc.select("h1.heading-large").text() shouldBe plaWithdrawWhatHappensInfoHeading
+    }
+
+    s"have a back link with text 'back' " in {
```

Comment:
Also needs to test the back link href location (using .attr("href") instead of .text for the href)